WPF/Practise.01.18/2020.01.18/View/MainWindow.xaml

```diff
@@ -0,0 +1,68 @@
+﻿<Window x:Class="_2020._01._18.MainWindow"
```

Comment:
Why do we have two windows with the same names in different namespaces?
Main window should be the only in the project.